googletest/include/gtest/internal/gtest-port.h

```diff
@@ -682,12 +682,12 @@ typedef struct _RTL_CRITICAL_SECTION GTEST_CRITICAL_SECTION;
 // A macro to disallow move operator=
 // This should be used in the private: declarations for a class.
 #define GTEST_DISALLOW_MOVE_ASSIGN_(type) \
-  type& operator=(type &&) = delete
+  type& operator=(type &&) noexcept = delete
```

Comment:
No show-stopper, but in which case could it useful to declare a deleted function `noexcept`? Just wondering...

Reply:
Well spotted, just move constructor should be noexcept.

Reply:
Thanks @trzecieu But the example at https://google.github.io/styleguide/cppguide.html#Copyable_Movable_Types does not even have its deleted move constructor (`NotCopyableOrMovable`) marked `noexcept`. Of course, the example could be out-of-date. But I'm just wondering, does `noexcept` have any effect on a deleted function. Anyway, no showstopper to me.